### ISSUE TITLE
HALUCINATION

### ISSUE THREAD

**Issue by swagatamr** [m-bain/whisperX#706]
how to reduce hallucination .
even after using VAD ,still hallucination is happening .
 Any other suggestions

**Comment by cleverestx** [comment]
Did you get anywhere with this?

**Comment by swagatamr** [comment]
Yes a little I was getting and for multilingual audios I was even getting more

**Comment by codenoid** [comment]
yeah, I was scared, because I only at home that night, but the transcription is kinda having output a valid sentence of my language, like, welcome, thankyou, I was scared
